redash/query_runner/clickhouse.py

```diff
@@ -87,25 +93,41 @@ def _get_tables(self, schema):
 
         return list(schema.values())
 
-    def _send_query(self, data, stream=False):
+    def _send_query(self, data, session_id=None, session_check=None):
         url = self.configuration.get("url", "http://127.0.0.1:8123")
+        timeout = self.configuration.get("timeout", 30)
```

Comment:
Should the timeout be configurable with an environment variable?

Reply:
The ClickHouse query runner had a timeout configuration option, and I had no reason to change it to the env variable. Should I introduce a new `REDASH_CLICKHOUSE_HTTP_TIMEOUT` env variable and remove the timeout option from the ClickHouse query runner configuration schema? (I am a bit worried about clients who would be required to reconfigure their Redash instance after an upgrade) I would be thankful for the explanation of the reasoning If you have time for that :)

Reply:
No you are correct. I wasn't thinking straight with my question 😱